### PR TITLE
Ewlj 79 podcast listing

### PR DESCRIPTION
**Pull request by aCyborg**
## Ticket(s)

**Github Issue**

N/A

**Jira Ticket**

- [EWLj-73: article listing prototype](https://issues.ama-assn.org/browse/EWLJ-73)

## Description

Create an podcast listing page prototype which includes:
* two-column-left--list layout

## To Test

- [ ] Spin up Patternlab
- [ ] Review the podcast listing
- [ ] Review the article listing
- [ ] Review two-column-left--list layout

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

**Comment by aCyborg** [comment]
Do not merge until PR #9 is merged.

**Review by gleroux02 — APPROVED** [review]
👍 once #9 is merged in.

**Pull request merged**
